### ISSUE TITLE
Autosized baseboard capacity not meeting the heating load

### ISSUE THREAD

**Issue by krishnan-gowri** [NREL/EnergyPlus#4347]
Reference ticket number: 9374 (Author: Daeho Kang, PNNL)
[EnergyPlus #9374]: Heating setpoints not being met for ZoneHVAC:Baseboard:RadiantConvective:Water

A user reported that baseboard heaters with autosized capacity do not meet all heating loads in the zone. 

PNNL investigated the issue and found that a portion of heat from the unit is delivered to the air in the zone. A linear relationship between baseboard heat outputs and heats to zone air was found in the literature. Based on this, a constant of 3.1 was approximated. Once it is applied, all heating loads were met, and other variables such as UA and water mass flow rate looked more reasonable.

In addition, the air mass flow rate is normalized in the calculation subroutine. When the differences between inlet water mass flow rate and standard water mass flow rate are too big and heating loads are fairly small, the current approach results in a very low air mass flow rate. To avoid this situation, the regression equation estimating air mass flow rate was used in the Calc subroutine.

**Comment by wfbuhl** [comment]
So the issue of switching 40% radiant to all convective and the baseboard still not meeting the temperature setpoint is solve by changing line 1127:

```
AirMassFlowRate = HWBaseboard( BaseboardNum ).AirMassFlowRateStd * ( WaterMassFlowRate / HWBaseboard( BaseboardNum ).WaterMassFlowRateStd );
```

to

```
AirMassFlowRate = HWBaseboard( BaseboardNum ).AirMassFlowRateStd * ( WaterMassFlowRate / HWBaseboard( BaseboardNum ).WaterMassFlowRateMax );
```

As the notes in the IDD say for Rated Water Mass Flow Rate: "Standard is I=B=R Rating document where all baseboards are rated at either 0.063 kg/s (1 gpm) or 0.252 kg/s (4 gpm).  It is recommended that users find data for the baseboard heater that corresponds to performance at 0.063 kg/s unless the flow rate is expected to be above 0.252 kg/s. If the flow rate is expected to be above 0.252 kg/s, this field should be 0.252 kg/s." The input for this field is 0.252 even though according to this note the second baseboard should be .063. But this doesn't help much with our problem. In fact there is no good reason to use WaterMassFlowRateStd in this formula for air mass flow rate. Given the rather ad hoc nature of this model, the air mass flow rate should be proportioned to the actual not the rated water mass flow rate.

So the change described fixes the convective problem, with the side issue that rated water mass flow rate is now never used. But it shouldn't be here in the first place unless we are interested in calculating the number of rated baseboards we need to meet the load.

As for the original input I think it is now working correctly The 40% radiant base board will take several days to warm the zone to the thermostat setpoint at winter design conditions, but it will eventually get there. Slow response? I don't think so given that all the walls are exterior, there are windows, and the outside temperature is -2C and there is no sun and no internal load. I think that with the above change things are fine. I don't know what to do with the useless "rated" field.

**Comment by mjwitte** [comment]
@wfbuhl Recent helpdesk ticket 10752, user is complaining that the water deltaT is too small when this baseboard operates.  I expect this fix will help that.  User's file posted in the 4347 defect folder.

**Comment by wfbuhl** [comment]
OK I'll take a look

**Comment by Myoldmopar** [comment]
Closed via #5214
